packages/app-desktop/gui/TaskNotification/TaskNotification.tsx

```diff
@@ -0,0 +1,87 @@
+import { useContext, useMemo, useEffect } from 'react';
+import { Dispatch } from 'redux';
+import { INotyfIcon, NotyfNotification } from 'notyf';
+import { themeStyle } from '@joplin/lib/theme';
+import { waitForElement } from '@joplin/lib/dom';
+import TaskUIService from '@joplin/lib/services/TaskUIService';
+import NotyfContext from '../NotyfContext';
+
+interface Props {
+	themeId: number;
+	dispatch: Dispatch;
+}
+
+export default (props: Props) => {
+	const notyfContext = useContext(NotyfContext);
+
+	const theme = useMemo(() => {
+		return themeStyle(props.themeId);
+	}, [props.themeId]);
+
+	const notyf = useMemo(() => {
+		const output = notyfContext;
+		output.options.types = notyfContext.options.types.map(type => {
+			if (type.type === 'success') {
+				// When type === 'success', type.icon is always of type object (INotyfIcon):
+				// https://github.com/caroso1222/notyf/blob/master/src/notyf.options.ts
+				(type.icon as INotyfIcon).color = theme.backgroundColor5;
+			} else if (type.type === 'error') {
+				// When type === 'error', type.icon is always of type object (INotyfIcon):
+				// https://github.com/caroso1222/notyf/blob/master/src/notyf.options.ts
+				(type.icon as INotyfIcon).color = theme.backgroundColor5;
+			}
```

Comment:
```suggestion
			if (type.type === 'success' || type.type === 'error') {
				// When type is 'success' or 'error', type.icon is always of type object (INotyfIcon):
				// https://github.com/caroso1222/notyf/blob/master/src/notyf.options.ts
				(type.icon as INotyfIcon).color = theme.backgroundColor5;
			}
```

It might be possible to combine these branches.